### DISCUSSION TITLE
Roll Call!

### DISCUSSION THREAD

**Discussion by malware-dev** [malware-dev/MDK-SE#314] [Announcements]
Hi guys! As you may or may not have noticed, Microsoft did me a dirty and broke the deployer of MDK in a way I can't fix. (if MDK still works for you... DO NOT update to Visual Studio 17.9!!!)

My problem right now, is that I don't actually know how popular this library is. Yes; it does have a fair number of stars and downloads... but... how many people actually use it?

Or; said in other words: Is it worth the effort?

MDK 2 _is_ underway, but I have a full-time job. To motivate myself, I need to have at least a _rough_ idea how many people would benefit from my work other than myself.  Therefore: Roll call.

Please react to this post (press the up-arrow). Let me know you want MDK2. Please let me know, in no uncertain terms, that I'm not wasting my time.

**Comment by malware-dev** [comment]
Yeah... I'm just not reaching people with these discussions, am I :D

Anyway, enough response to be worth it I guess :D Thanks, folks.

**Comment by largenumberhere** [comment]
I realize this is 2 months late, but I feel it would be inconsiderate to not share some appreciation here.
I suspect the 0 replies here is just a quirk of your users - I personally, learnt about the MDK-1 through discord long before I really knew how to use Github.

The space engineers MDK was what allowed me to get confident in a real programming language. I learnt so much with the power and convenience of the tool. Since space engineers has a passable but professional API and allows such complex logic, I was able to see immediate and fun results of changes I made (eg: a ship crashes, a ship spins in circles then crashes, a ship explodes). I would have given up very early on if I couldn't use a context-aware IDE to help manage the complexity and a way to deploy scripts that was automated. Space engineers is uniquely challenging and interesting but also streamlined to script for thanks to MDK-1.

You're not wasting your time, I think MDK-1 is an amazing tool and I really want other people like me to have this opportunity in future. So long as your time, health, etc permits, please continue to maintain MDK-1 as best you can until MDK-2 is fully released.

Thank you

**Reply by malware-dev** [reply]
Thank you. Rest assured, I am not abandoning anything.
